sdk/tables/Azure.Data.Tables/tests/samples/Sample6_TransactionalBatchAsync.cs

```diff
@@ -32,7 +32,9 @@ public async Task TransactionalBatchAsync()
 
             #region Snippet:BatchAdd
             // Create a list of 5 entities with the same partition key.
-            //@@string partitionKey = "BatchInsertSample";
+            #if SNIPPET
+string partitionKey = "BatchInsertSample";
```

Comment:
@pakrym FYI, this is a snippet spacing bug. Without removing the indents here, the md added extra indents.

Reply:
Try removing the indent on the `            #if SNIPPET` line